src/convex/ConvexBoosterController.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {IController} from "../core/IController.sol";
+
+interface IBooster {
+    function poolInfo(uint256) external view returns(address, address, address, bool, address);
+}
+
+contract ConvexBoosterController is IController {
+
+    address public immutable BOOSTER;
+
+    /// @notice deposit(uint256,uint256)
+    bytes4 constant DEPOSIT = 0xe2bbb158;
+
+    constructor(address booster) {
```

Comment:
we can use the target address rather than specifying the booster here

Reply:
Constants save gas and you don't need the target address once you're at this point so this should be better right?

Reply:
The contract relies on one convex booster, if we have multiple convex boosters we will have to deploy multiple controllers. If convex change their booster address we have to again deploy a new controller. Using target allows us to rely only on config.

Reply:
> If we have multiple convex boosters 

Why/When would they need multiple boosters?

>  If convex change their booster address

It's a non-upgradeable deployment which would only change for fixes that require the entire protocol to be migrated.

Reply:
They way I see it is that each controller is meant to work for a contract of that type, and hence should work for all contracts of that type, this works but requires a redeployment for each contract, but I do see your point on gas plus the fact that these contracts won't change often.